Config/config.php

```diff
@@ -38,4 +38,34 @@
     | Expressed in bytes
     */
     'max-total-size' => 1000000000,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Define which assets will be available through the asset manager
+    |--------------------------------------------------------------------------
+    | These assets are registered on the asset manager
+    */
+    'media-partial-assets' => [
+        // Styles
+        'asgard.css' => ['theme' => 'css/asgard.css'],
```

Comment:
I mean that it's already getting loaded on the page via this: https://github.com/AsgardCms/Core/blob/master/Config/core.php#L124 Wouldn't this require it twice ?